### ISSUE TITLE
When Narrator/NVDA is on user needs to press ESC key two times to close command palette

### ISSUE THREAD

**Issue by LadyNaggaga** [Microsoft/monaco-editor#1167]
.NET Online REPL  is having the following accessibility issue.  Please see this [issue](https://github.com/dotnet/try/issues/120) . I have also included the details below 
> **Environment Details**
> https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> **Browser:** Edge (41.16299.402.0)
> Chrome (67.0.3396.87)
> **Screen Reader:** Narrator
> NVDA
> 
> **Additional Details:**
> MAS Violated: MAS1.3.2, MAS2.1.1
> 
> **Repro steps:**
> 
> 1. Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
> 2. Tab till “Enter focus mode” and press enter
> 3. Type any code and press F1 and on the Screen reader
> 4. Check whether user is able to close the Collaboration list on pressing Esc only once when screen reader is on
> 
> **Actual Result:-**
> When screen reader is on user needs to press ESC key two time to close command palette
> 
> **Expected Result:-**
> User should be able to close the list items on pressing the dropdown only once. Command to perform any operation should be changed when Screen reader is on.
> 
> **User Impact:-**
> Screen Reader users find difficulty in using application if user needs to operate controls twice to perform an action.
> 
> **MAS Reference:-**
> [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B1ee227a7-e3c5-47b8-ab18-48ca8595077b%7D)
> 
> [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D)
> 
> [When screen reader is on user needs to press ESC key two time to close command palette.pptx](https://github.com/dotnet/try/files/2191424/When.screen.reader.is.on.user.needs.to.press.ESC.key.two.time.to.close.command.palette.pptx)

**Comment by bpasero** [comment]
@chrmarti I can reproduce this also in VSCode, steps:
* run NVDA on Windows
* open a picker (e.g. language picker)
* press ESC
* nothing happens
* press ESC again
* now the picker closes

Any clues or recent changes you can think of?

**Comment by chrmarti** [comment]
This appears to be 'as designed': https://github.com/Microsoft/vscode/issues/58000#issuecomment-418958024 and https://github.com/nvaccess/nvda/issues/4428

**Comment by bpasero** [comment]
Thanks.
